content/en/hosting/4.x/data-migration-k8s.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Migration from CHT 3.x to CHT 4.x in Kubernetes"
+linkTitle: "K8s Data Migration to 4.x"
+weight: 1
+aliases:
+  - /apps/guides/hosting/4.x/k8s-data-migration
+description: >
+  Guide to migrate existing data from CHT 3.x to CHT 4.x in Kubernetes environments
+relatedContent: >
+---
+
+The hosting architecture differs entirely between CHT-Core 3.x and CHT-Core 4.x. When both versions are running in Kubernetes, migrating data requires specific steps using the `couchdb-migration` tool. This tool interfaces with CouchDB to update shard maps and database metadata.
```

Comment:
Can you add a link to the CouchDB Migration tool? That way folks can read up on it before using it.